crates/client/data-availability/src/ethereum/config.rs

```diff
@@ -0,0 +1,64 @@
+use std::fs::File;
+use std::path::PathBuf;
+
+use serde::Deserialize;
+
+use crate::DaMode;
+
+pub const DEFAULT_ETHEREUM_NODE: &str = "127.0.0.1:8545";
+pub const DEFAULT_SEQUENCER_KEY: &str = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
```

Comment:
Where does this value come from?

Reply:
// default key derived from starting anvil as follows:
// anvil -b 5 --config-out $BUILD_DIR/anvil.json
// PRE_PRIVATE=$(jq -r '.private_keys[0]' $BUILD_DIR/anvil.json)

added as comment